src/pyuvsim/mpi.py

```diff
@@ -51,11 +51,13 @@ def start_mpi(block_nonroot_stdout=True):
         Redirect stdout on nonzero ranks to /dev/null, for cleaner output.
 
     """
+    do_once = False
     global world_comm, node_comm, rank_comm, rank, Npus
-    if not MPI.Is_initialized():
+    if not MPI.Is_initialized():  # pragma: no cover
```

Comment:
This isn't covered because we use `mpi-pytest` to run our tests which initializes MPI before we get here.